dspace/src/main/config/build.xml

```diff
@@ -947,94 +947,89 @@ You may manually install this file by following these steps:
         <antcall target="update_geolite" />
     </target>
 
-    <!-- Check if any Solr indexes need updating to the version of Solr/Lucene we are using. -->
+    <!-- Check if any Solr indexes need updating to the version of
```

Comment:
Is this upgrade working now?  It is good to see that this could work. I presume new documents inherit the the version from the schema/config files but existing index documents are aware of their old versions.

Reply:
Good point.  This is untested.  The current focus is on fresh_install, which won't be running this target.  Arguably I should remove this change to a separate PR for supporting upgrade.

Reply:
I've now inadvertently tested the trivial case:  when the cores are already v7 this target correctly does not try to upgrade them.  Actually upgrading older cores remains untested.